Userland/DevTools/HackStudio/Dialogs/NewProjectDialog.h

```diff
@@ -21,14 +21,15 @@ namespace HackStudio {
 class NewProjectDialog : public GUI::Dialog {
     C_OBJECT(NewProjectDialog);
 
+    virtual ~NewProjectDialog() override = default;
```

Comment:
This change is not needed, please revert this file

Reply:
Thank you, I had done that for another thing, I will revert it.